Makefile

```
@@ -0,0 +1,25 @@
+## TODO this is just a rudimentary sceleton with a couple of commands that need to be implemented propery
+
+IMAGE_NAME = camptocamp/nominatim
+CONTAINER_NAME = nominatim
```

I propose to use `?=` to conditionnally assign the variable if not set, but so we can set it outside of makefile in envvar to build images with other names. I also suggest to add `IMAGE_TAG` for the same reason
```suggestion
IMAGE_NAME ?= camptocamp/nominatim
IMAGE_TAG ?= latest
CONTAINER_NAME ?= nominatim
```